core-services/prow/02_config/openshift-eng/art-tools/_pluginconfig.yaml

```diff
@@ -55,3 +55,9 @@ plugins:
     - wip
     - yuks
     - approve
+branch_cleaner:
```

Comment:
pretty sure we want to limit the plugin to our repo, and i don't think this does?